src/components/pages/manage/productions/create/index.js

```diff
@@ -50,19 +50,19 @@ const Create = () => {
     await refetchSuggestedEvents();
   };
 
-  const { mutate: skipSuggestedEvents } = useSkipSuggestedEvents({
+  const skipSuggestedEventsMutation = useSkipSuggestedEvents({
```

Comment:
To go a step further, we can just call this `skipSuggestedEvents` too, no need to suffix to it with 'Mutation'.

But we can do that in a different PR, let's merge this one :)